test/snippet/search/Algorithm/search.cpp

```diff
@@ -0,0 +1,21 @@
+#include <vector>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/search/algorithm/search.hpp>
+#include <seqan3/search/fm_index/all.hpp>
+
+int main()
+{
+    using seqan3::operator""_dna4;
+    std::vector<std::vector<seqan3::dna4>> genomes{"CGCTGTCTGAAGGATGAGTGTCAGCCAGTGTA"_dna4,
+                                                   "ACCCGATGAGCTACCCAGTAGTCGAACTG"_dna4,
+                                                   "GGCCAGACAACCCGGCGCTAATGCACTCA"_dna4};
+    std::vector<std::vector<seqan3::dna4>> queries{"GCT"_dna4, "ACCC"_dna4};
+
+    seqan3::bi_fm_index index{genomes};                                // build the index
```

Comment:
Is somewhere mentioned in this context that there is a `fm_index` too? Otherwise, since this is global documentation of search, add a comment with something like `You can also use other indices, e.g. the seqan3::fm_index` which then can be edited once we have the kemr_index and the dream index.